From c8c114333da972e2cacbe21da161458c39b760a3 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Wed, 30 Sep 2020 09:26:29 +0300
Subject: [PATCH] [SW-21029] goya2: ppboot: add support for bss

This patch adds support for bss section in ppboot
so that global variables usage is possible in ppboot.
In the next patch, we would take care of clearing the
BSS section.

Change-Id: Iedd66cb3d6b3e60c69ac3c203e4be2778a7f9988
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/goya2/ppboot.lds.S b/goya2/ppboot.lds.S
index e078b30..1e5be88 100644
--- a/goya2/ppboot.lds.S
+++ b/goya2/ppboot.lds.S
@@ -20,6 +20,13 @@
 		* (.text*)
 	} >.sram
 
+	.bss : {
+		. = ALIGN(8);
+		__bss_start = . ;
+		*(.bss*)
+		__bss_end = . ;
+	} > .sram
+
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
